robot_controll/main.cpp

```diff
@@ -20,6 +19,7 @@
 #include "GpsThread.h"
 //#include "PhoneThread.h"
 #include "Coordinate.h"
+#include "DataTypes.h"
```

Comment:
@pavelpetrovic This is necessary for the `Ll` class used below (it represents longitude/latitude).